packages/devtools_app/lib/src/screens/debugger/codeview.dart

```diff
@@ -544,6 +581,174 @@ class _CodeViewState extends State<CodeView>
   }
 }
 
+class ProfileInformationGutter extends StatelessWidget {
+  const ProfileInformationGutter({
+    required this.scrollController,
+    required this.lineOffset,
+    required this.lineCount,
+    required this.sourceReport,
+  });
+
+  final ScrollController scrollController;
+  final int lineOffset;
+  final int lineCount;
+  final ProcessedSourceReport sourceReport;
+
+  static const totalTimeTooltip =
+      'Percent of time that a sampled line spent executing its own\n code as '
+      'well as the code for any methods it called.';
+
+  static const selfTimeTooltip =
+      'Percent of time that a sampled line spent executing only its own code.';
+
+  @override
+  Widget build(BuildContext context) {
+    final theme = Theme.of(context);
+    final gutterWidth = CodeView.assumedCharacterWidth * 10 + denseSpacing;
+    return Container(
+      width: gutterWidth,
+      decoration: BoxDecoration(
+        border: Border(right: defaultBorderSide(theme)),
+        color: Theme.of(context).titleSolidBackgroundColor,
+      ),
+      child: Stack(
+        children: [
+          Column(
+            children: [
+              Container(
+                height: CodeView.rowHeight,
+                child: Row(
+                  crossAxisAlignment: CrossAxisAlignment.stretch,
+                  mainAxisAlignment: MainAxisAlignment.spaceAround,
+                  children: const [
+                    Expanded(
+                      child: DevToolsTooltip(
+                        message: totalTimeTooltip,
+                        child: Text(
+                          'Total %',
+                          textAlign: TextAlign.center,
+                        ),
+                      ),
+                    ),
+                    SizedBox(width: denseSpacing),
+                    Expanded(
+                      child: DevToolsTooltip(
+                        message: selfTimeTooltip,
+                        child: Text(
+                          'Self %',
+                          textAlign: TextAlign.center,
+                        ),
+                      ),
+                    ),
+                  ],
+                ),
+              ),
+              const Divider(
+                height: 0,
+              ),
```

Comment:
what is the purpose of a divider of height 0? does this actually add a border?

Reply:
`height: 0` effectively removes the padding from around the divider. The divider is still rendered, just much closer to the widget above.

Reply:
Can you use `OutlineDecoration.onlyBottom` instead?

Reply:
It's not visually identical to a `Divider`, so we'd have two different line styles for the header horizontal divider and the vertical divider separating the two columns.